code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -805,22 +805,22 @@
 	drink_name = "Amasec"
 	drink_desc = "Always handy before COMBAT!!!"
 
-/datum/reagent/consumable/ethanol/neurotoxin
-	name = "Neuro-toxin"
```

Comment:
IMHO, they need unique names. Too many times people confuse one variant of neurotoxin for the other.